tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -285,11 +287,20 @@ private void addConfToResources(Configuration conf, String confAddress, String c
     }
   }
 
-  private void logTrackingAndRMUrls(ApplicationReport report) {
+  private void logTrackingAndAMRMUrls(ApplicationReport report) {
     LOG.info("URL to track running application (will proxy to TensorBoard once it has started): "
              + report.getTrackingUrl());
     LOG.info("ResourceManager web address for application: "
         + Utils.buildRMUrl(yarnConf, report.getApplicationId().toString()));
+    try {
+      ContainerReport amContainerReport = yarnClient.getContainers(report.getCurrentApplicationAttemptId())
+          .stream()
+          .min(Comparator.comparingLong(x -> x.getContainerId().getContainerId()))
+          .orElseThrow(YarnException::new);
+      LOG.info("AM log url: " + amContainerReport.getLogUrl());
```

Comment:
`Driver (application master) log url:`

Will be more intuitive for users

Reply:
Fixed